cozy/pubspec.yaml

```diff
@@ -12,7 +12,7 @@ dependencies:
 
   linagora_design_flutter:
     git:
-      url: https://github.com/linagora/linagora-design-flutter.git
+      url: git@github.com:linagora/linagora-design-flutter.git
```

Comment:
why we need this?

Reply:
It is required from the `flutter_emoji_mart` library